### ISSUE TITLE
Missing Letter Icon if email starts with a image

### ISSUE THREAD

**Issue by joeytroy** [boukestam/inbox-in-gmail#41]
Not sure this is even worth complaining about but I am just thinking maybe a user may have an issue. Because Chess.com uses a pawn image before their email it doesn't render the letter C

![image](https://user-images.githubusercontent.com/1558465/55915737-8cb74d80-5ba7-11e9-9e3e-8ffb94e7ffc5.png)

**Comment by russelldc** [comment]
@g60madman Could you email that to me? I don't think forwarding would work well, maybe just copy the content of the email body.

**Comment by joeytroy** [comment]
@russelldc sent.

**Comment by russelldc** [comment]
@g60madman Unfortunately the .eml file isn't much help. I guess I could try getting an email from them myself? Is that the only email you have that's doing that? Are you sure it's because the email "starts with an image". Also, are you calling that chess unicode character (♟) an image, or do you mean the image in the email body?

**Comment by russelldc** [comment]
@g60madman Could you copy and paste the HTML element of that email in the inbox list?

**Comment by russelldc** [comment]
OH. Sorry, I understand now. I forgot how that avatar code worked, grabbing the first letter of the user's name.

**Comment by russelldc** [comment]
I've got a fix. This problem was also affecting any contact who's name does not start with A-Z, including, for example: '23AndMe'.

I've pushed it to the inbox-in-gmail/inbox-in-gmail fork.

<img width="175" alt="Screen Shot 2019-04-11 at 12 09 00 AM" src="https://user-images.githubusercontent.com/5100126/55919515-2d306080-5bee-11e9-89bd-76c2bdeb9569.png">

Forced the ♟ character:

<img width="321" alt="Screen Shot 2019-04-11 at 12 03 54 AM" src="https://user-images.githubusercontent.com/5100126/55919526-38838c00-5bee-11e9-800f-8c64eb97050f.png">

Can't do anything about the odd spacing, for some reason the ♟ character has white space to the right of it built-in.
